yuniql-cli/BaseOption.cs

```diff
@@ -10,7 +10,7 @@ public class BaseOption
 
         //yuniql <command> -d | --debug
         [Option('d', "debug", Required = false, HelpText = "Print debug information including all raw scripts.")]
-        public bool IsDebug { get; set; }
+        public bool? IsDebug { get; set; } 
```

Comment:
I'm curious why you think this has to be changed. I'm afraid it may introduce bugs not directly related to feature we're implementing. :)

Reply:
The raison why I made it nullable is I want to differentiate between two states : 

1. IsDebug is false && the value wasn't specified by the user (default value for boolean is False [doc)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values)
2. When the user specify --debug false
**In both cases IsDebug set to false bit the source of this false is different**